ontology/yaml/resources/HVAC/entity_types/TANK.yaml

```diff
@@ -16,6 +16,30 @@
 ### Canonical Types  ###
 ########################
 
+TK_SS_LVLHL:
+  guid: "f6beac97-8d0b-4668-a7a4-0352b7205ba2"
+  description: "Tank with high and low water level monitoring."
+  is_canonical: true
+  implements:
+  - SS
```

Comment:
What is the SS referring to here? (What is the run command and status for?)

Reply:
Apologies no need for SS here. Thanks, Ciaran